### DISCUSSION TITLE
I want DevTools to help me discover media that needs subtitling or captions

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#191] [Wants]
It’s hard to tell—without looking at the markup—which video elements are missing captioning tracks. It would be cool if DevTools could highlight that as an issue for me. Double bonus if some cool AI could enable me to auto-generate a first pass at building a caption file.

<hr>

https://webwewant.fyi/wants/1/

**Comment by snarfed** [comment]
test comment. @aarongustafson mind upvoting, reacting, and replying? I'm trying to get discussion notifications so I can see what they look like in the API.

**Reply by aarongustafson** [reply]
Happy to 🙂

**Reply by aarongustafson** [reply]
Replied, reacted, and upvoted
